### PR TITLE
Added Content-encoding to gzip

### PR DESCRIPTION
**Pull request by rlajous**
## Summary
- Added Content-Encoding metadata by a command if you wanna deploy a gziped file

### CODE REVIEW

File: script/s3.js [hunk 1]
```diff
@@ -71,6 +75,15 @@ const emptyBucket = (bucketName, callback) => {
 
 const isMainFile = file => file === "index.html";
 
+const isGzip = metaData => {
+  if (gzip){
+    return {...metaData, ContentEncoding: 'gzip'};
+  }
+  else {
```
Comment (damfinkel) [inline script/s3.js]: else in the previous line

File: script/s3.js [hunk 2]
```diff
@@ -71,6 +75,15 @@ const emptyBucket = (bucketName, callback) => {
 
 const isMainFile = file => file === "index.html";
 
+const isGzip = metaData => {
+  if (gzip){
+    return {...metaData, ContentEncoding: 'gzip'};
```
Comment (damfinkel) [inline script/s3.js]:
```suggestion
    return { ...metaData, ContentEncoding: 'gzip' };
```

File: script/s3.js [hunk 3]
```diff
@@ -71,6 +75,15 @@ const emptyBucket = (bucketName, callback) => {
 
 const isMainFile = file => file === "index.html";
 
+const isGzip = metaData => {
+  if (gzip){
```
Comment (damfinkel) [inline script/s3.js]:
```suggestion
  if (gzip) {
```

File: script/s3.js [hunk 4]
```diff
@@ -71,6 +75,15 @@ const emptyBucket = (bucketName, callback) => {
 
 const isMainFile = file => file === "index.html";
 
+const isGzip = metaData => {
```
Comment (damfinkel) [inline script/s3.js]: I'd rename this to `getMetadataObject`. If named `isGzip` it appears to be a function returning a boolean

**Review by damfinkel — CHANGES_REQUESTED** [review]

**Review by damfinkel — DISMISSED** [review]

**Review by SKOLZ — DISMISSED** [review]

File: script/s3.js [hunk 5]
```diff
@@ -15,10 +15,14 @@ const colors = {
 
 let relativePath, env = "development";
 let outputPath = '';
+let gzip = false;
```
Comment (SKOLZ) [inline script/s3.js]: I'd remove the if in the following line and set this to:
```
const gzip = args.gzip || args.g;
```

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by SKOLZ — APPROVED** [review]

**Review by damfinkel — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**